File: sample-videochat-webrtc/sample-videochat-webrtc/Components/Settings/VideoSettingsViewController/SwitchItemModel.m
//
//  SwitchItemModel.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "SwitchItemModel.h"
#import "SettingSwitchCell.h"

@implementation SwitchItemModel

- (Class)viewClass {
    
    return [SettingSwitchCell class];
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Settings/SwitchItemModel.swift
//
//  SwitchItemModel.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/11/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation

class SwitchItemModel: BaseItemModel {
    //MARK: - Properties
    var on = false
    
    //MARK: - Overrides
    override func viewClass() -> AnyClass {
        return SettingSwitchCell.self
    }
}
